_docs/ops/contingency-plan.md

```diff
@@ -86,14 +86,56 @@ The Incident Commander declares that recovery efforts are complete and notifies
 
 Next, the team schedules an internal postmortem to discuss the event. This is the same as the [security incident retrospective process]({{ site.baseurl }}{% link _docs/ops/security-ir.md %}#initiate). Then we write a public postmortem and post it on StatusPage for users. We should also discuss with our JAB representatives whether they need additional information.
 
+## Internal dependencies
+
+Continued operation of cloud.gov relies on our people, but also on access to internal diagnostics/repair through our `jumpbox`. We also have
+internal GSA dependencies that may be impacted if a GSA COOP (Continuity of Operations Plan) is in effect. Resources for the GSA/TTS COOP plan include:
+
+* [TTS Devolution of Operations Playbook (Jan 2020)](https://drive.google.com/file/d/1sfSVhVPBteobeqhqnyb152z1PqboOKJI/view)
+* [TTS Google COOP Site](https://sites.google.com/a/gsa.gov/continuity/home/coming-soon---customized-hsso-pages/ocsit)
+* [TTS-wide Emergency Contact Spreadsheet](https://docs.google.com/spreadsheets/d/1-Hhv5S0M03W4JY1k-CM5vxrZildHUdMaxAoqLTWF2Ts/edit#gid=1295161336)
+* [(deprecated) TTS Emergency Personnel Roster](https://docs.google.com/spreadsheets/d/1ITXXOu2IAntV8r6snIYHyXlRt1MgIlxpr-GudDP8tqQ/edit#gid=0)
+
+### Leadership
+
+The director and deputy director own administrative credentials critical to long-term
+platform operation and customer support. The 
+[recovery process for accessing those credentials](https://github.com/cloud-gov/internal-docs/blob/master/runbooks/contingency.md#leadership-credentials)
+ are non-public (to avoid potential social engineering). These credentials would be delegated to a GSA interim cloud.gov system owner if leadership is
+ abruptly unavailable for 3 or more working days, or if COOP plan is otherwise in effect.
+
+### Engineer staffing
+
+If the number of available platform engineers is 
```

Comment:
The language in this section is a concern. The details are too prescriptive and usurps the authority of the deputy director, director, and TTS leadership; specifically, leadership must maintain the authority determine staffing requirements, reaction to limited staffing, and tasking of staff.

Reply:
How about prefacing it with:

> In the event that the Director, Deputy Director, or TTS leadership are unavailable to make an independent determination, then if the number of engineers _etc,, etc_...